lib/Dialect/TTIR/IR/TTIROps.cpp

```diff
@@ -126,6 +126,36 @@ ::mlir::LogicalResult mlir::tt::ttir::TransposeOp::verify() {
   return success();
 }
 
+::mlir::LogicalResult mlir::tt::ttir::ReshapeOp::verify() {
```

Comment:
Do we want to check if there is an appropriate number of elements in the input/output tensor? 

E.g. [1, 1, 3, 3] can be reshape into [1, 1, 1, 9], but can't into [1, 1, 1, 10].

Reply:
Done